python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -30,7 +39,10 @@ def list_of_strings(arg):
 parser = argparse.ArgumentParser(parents=[kserve.model_server.parser])
 
 parser.add_argument(
```

Comment:
for #3580 mentioned in the PR, `--trust-remote-code` should be supported here for HF server.